job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -386,8 +465,7 @@ definitions:
         format: int64
         description: The time the job status or progress last changed
       status:
-        type: string
-        description: String value of the job status enum, possible values are Completed, Active, Waiting, Failed, Paused and Cancelled.
+        $ref: "#/definitions/job-status"
```

Comment:
This change was not strictly necessary but felt it was cleaner to use an enum vs a string here now for the response from the existing endpoints, as thas what the new status endpoint is using.

This does mean a breaking change to the job-service-client though, as the generated client code will have to handle a `JobStatus` enum now rather than a string:

https://github.com/JobService/job-service-client/pull/3/files#diff-d5ca635f7b96abc24783be438f79d8ec92d70a7a32b64ae7b2049d159697c017R11

The alternative is to just leave it as a String, and use a String rather than an enum for the new status endpoint as well.